hack/tools/release/notes.go

```diff
@@ -318,3 +327,27 @@ func getCommitHashFromNewTag(newTag string) (string, error) {
     }
 }
 
+func trimPrereleasePrefix(version string) string {
+	if idx := strings.Index(version, "-"); idx != -1 {
+		return version[:idx]
+	}
+	return version
+}
+
+func wrapAndPrintln(input string, maxLength int) {
```

Comment:
This cannot handle the PR number in the end right ?  It cuts it off we go too long. If so, this is not ok.

We can maybe just put "markdownlint-ignore line-length"  with the  no-inline-html.